.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+version: '3'
```

Comment:
When I follow your blog post, mine generates with a different version. Does it matter?

```suggestion
version: '3.8'
```

Reply:
Old habits die hard for me, for maintainability of the docker-compose file it's best to just keep the full version used, thanks!